### DISCUSSION TITLE
How to unregister from push notifications on Android

### DISCUSSION THREAD

**Discussion by alubeck** [invertase/react-native-firebase#4812] [Q&A] [answered]
Hi,

for iOS have these beautiful methods `(un)registerDeviceForRemoteMessages()` to (un)subscribe from push notifications in an app. But how can I achieve something like that for android? How to temporarily pause from getting push notifications on Android? Any help appreciated, thanks in advance.

**Comment by mikehardy** [comment, marked as answer]
Cycle the token I think.

**Reply by alubeck** [reply]
Thanks very much @mikehardy, that worked out. My mind went blank. 😆

**Reply by mikehardy** [reply]
Ironically, my work app still has the same issue when users log out, the new user can still receive the old user's FCM messages because...I'm not cycling the token 😉 . Never enough time to fix everything 😅
